readme.md

```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
-as you apply for jobs. 
+Instructors will not be grading this project. We want you to use this final project as your magnum opus as you apply for jobs.
 
-If you would like instructor feedback after your project has been completed, please **submit a link to your Github repository as a Git issue to the [`pbj-project4-gallery`](https://github.com/ga-dc/pbj-project4-gallery) repository**. If you do so **by 6:00pm on Friday**, we will do our best to have feedback returned to you by Monday morning. If you do so afterward, please just Slack an instructor from whom you'd like feedback to let them know.
+If you would like instructor feedback after your project has been completed, please **submit a link to your Github repository as a Git issue to the [`wdi-project4-gallery`](https://github.com/ga-dc/wdi-project4-gallery) repository**. If you do so **by 6:00pm on Friday**, we will do our best to have feedback returned to you some time the following week. If you do so afterward, please just Slack an instructor from whom you'd like feedback to let them know.
 
-As always, instructors will be available for 1-1 appointments and support. Please **submit technical issues to the `pbj-project4` repository as Git issues**. To schedule an appointment, please Slack the relevant instructor.
+As always, instructors will be available for 1-1 appointments and support. Please **submit technical issues to the `project4` repository as Git issues**. To schedule an appointment, please Slack the relevant instructor.
 
 ---
 
 ### Presentations
-
 These will be "Science Fair-style" on Friday.
 
 ---
-
-### Dates and times
-
-- Monday
-  - 2:00 - 3:00pm: You and your squad will scrum with your instructor, followed by optional code reviewing with your squadmates. Your squad instructor will take attendance.
-- Wednesday
-  - 2:00 - 3:00pm: You and your squad will scrum with your instructor, followed by optional code reviewing with your squadmates. Your squad instructor will take attendance.
-- Friday
-  - 12:00: Attendance will be taken. You and another student will pair to peer-review each others' code.
-  - 12:30 Lunch is provided!
-  - 3:00: Squads Ada, Bash, and C will present at the Science Fair.
-  - 3:30: Break
-  - 3:45: Squads Dart, Elixir, and Fortran will present at the Science Fair.
-  - 4:15: Break
-  - 4:30: The class will debrief as a whole.
-  - 5:30: The start of the rest of your lives! (And Happy Hour!)
+### Schedule - Dates and times
+#### Fri - 8 Jan
+|time| what's happening|
+|---|---|
+|9:00| Project 4 Kickoff. Pick whichever classroom you want! |
+|9:00 - 11:00| Project Brainstorm - user stories, some initial sketches/wireframing, possible ERD's|
+|11:30| Squad Scrum - Bounce stories off squad and instructor
+|11:00 - 4:00| Work on projects |
+|4:00| Feed BACK(robin's activity)|
+|5:00| Squad Scrum - Discuss project 1 |
+
+#### Mon - 11 Jan
+Schedule 15 minute appt with instructor. Attendance will be taken at appt by squad leader.
```

Comment:
Phrasing sounds like the scheduling of the meeting happens Monday. I assume the intent is that the 15 minute appt (which was scheduled Friday or over the weekend) will occur on Monday.